src/lib/modules/evaluate/evaluate.go

```diff
@@ -0,0 +1,92 @@
+package evaluate
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"time"
+
+	"github.com/go-ping/ping"
+)
+
+// Ping icmp requests to target.
+type Ping struct {
+	Min  time.Duration `json:"min"`
+	Mean time.Duration `json:"mean"`
+	Max  time.Duration `json:"max"`
+}
+
+// Jitter is the mean variation between consecutive ping operations.
+type Jitter struct {
+	Value time.Duration `json:"jitter"`
+}
+
+// Response contains evaluations of response delay and length, calculated during monitoring of an API.
+type Response struct {
+	Delay  time.Duration `json:"delay"`
+	Length int           `json:"length"`
+	Size   []byte        `json:"size"`
+}
+
+// Machine evaluates calculations related to ping and jitter.
+func Machine(host string) (*Ping, *Jitter, error) {
+	pinger, err := ping.NewPinger(host)
```

Comment:
Can we come up with a better name than `pinger` like PingManager or something, sounds weird to me

Reply:
nvm, this should come from the library, so can't do much about it :3